includes/class-dispatcher.php

```diff
@@ -99,11 +99,12 @@ private static function send_activity_to_followers( $activity, $actor_id, $outbo
 		/**
 		 * Filters the list of inboxes to send the Activity to.
 		 *
-		 * @param array    $inboxes  The list of inboxes to send to.
-		 * @param int      $actor_id The actor ID.
-		 * @param Activity $activity The ActivityPub Activity.
+		 * @param array    $inboxes     The list of inboxes to send to.
+		 * @param int      $actor_id    The actor ID.
+		 * @param Activity $activity    The ActivityPub Activity.
+		 * @param \WP_Post $outbox_item The WordPress object.
 		 */
-		$inboxes = apply_filters( 'activitypub_send_to_inboxes', array(), $actor_id, $activity );
+		$inboxes = apply_filters( 'activitypub_send_to_inboxes', array(), $actor_id, $activity, $outbox_item );
```

Comment:
what about sending a `do_action` with the result of the `safe_remote_post` in line 113? this way we could even track the response!?!

Reply:
Took a stab at it in [499c902](https://github.com/Automattic/wordpress-activitypub/pull/1229/commits/499c9020fefe951152fd1d7aa0f5c28b15762910).